src/Simplex/Chat.hs

```diff
@@ -525,7 +526,18 @@ processChatCommand = \case
         withStore' $ \db -> deleteGroupConnectionsAndFiles db user gInfo members
         withStore' $ \db -> deleteGroupItemsAndMembers db user gInfo members
         withStore' $ \db -> deleteGroup db user gInfo
+        forM_ contactIds $ \ctId ->
+          deleteUnusedContact ctId `catchError` \_ -> pure ()
         pure $ CRGroupDeletedUser gInfo
+      where
+        deleteUnusedContact contactId = do
+          ct <- withStore $ \db -> getContact db user contactId
+          unless (directContact ct) $ do
+            ctGroups <- withStore' $ \db -> getContactGroupProfiles db user ct
```

Comment:
why do we need to load the actual profiles instead of just checking they exist? This could be a function returning a boolean.

Reply:
it's what we'll need / close to that for getContactGroups api we for displaying in chat info